winml/lib/Api.Ort/OnnxruntimeEngine.cpp

```diff
@@ -1259,7 +1259,6 @@ const WinmlAdapterApi* OnnxruntimeEngineFactory::UseWinmlAdapterApi() {
 HRESULT OnnxruntimeEngineFactory::GetOrtEnvironment(OrtEnv** ort_env) {
   RETURN_IF_FAILED(EnsureEnvironment());
   RETURN_IF_FAILED(environment_->GetOrtEnvironment(ort_env));
-  RETURN_HR_IF_NOT_OK_MSG(ort_api_->EnableTelemetryEvents(*ort_env), ort_api_);
```

Comment:
Why is this removed?

Reply:
because it's on by default now, so winml doesn't need to turn it on.